src/usr/local/www/vpn_ipsec_mobile.php

```diff
@@ -76,7 +83,8 @@
 	$pconfig['wins_server2'] = $a_client['wins_server2'];
 	$pconfig['pfs_group'] = $a_client['pfs_group'];
 	$pconfig['login_banner'] = $a_client['login_banner'];
-
+	$pconfig['radius_ip_priority_enable'];
```

Comment:
This does not look like it would do anything. Does it serve a purpose?

Maybe you intended to set this value based on its equivalent in `$a_client`?